### ISSUE TITLE
Does this still work?

### ISSUE THREAD

**Issue by YohannesTz** [aykuttasil/CallRecorder#63]
Just asking if this library still works on older and newer devices. I have seen many places that are saying Google has removed those types of APIs.

**Comment by FerretFerret** [comment]
Probably not..Google restricted access to Accessibility Services and Contacts as far as Phone Call Recorders are concerned.
  They did this in 2022.
The result is you can only record yourself. You will only hear one party,and there will be no phone numbers listed.

**Comment by savan15** [comment]
> Probably not..Google restricted access to Accessibility Services and Contacts as far as Phone Call Recorders are concerned. They did this in 2022. The result is you can only record yourself. You will only hear one party,and there will be no phone numbers listed.

Yes as you say google is restricted above things. but let's asume here we need to just for our test/educational purpose we record audio of both side so how to go with that. 

As currenlty i have able to capture only one side of voice call person audio record not able to record both side audio record.

also there are some apps on google play store they have successfully record audio of both side.

**Comment by lifei** [comment]
> > Probably not..Google restricted access to Accessibility Services and Contacts as far as Phone Call Recorders are concerned. They did this in 2022. The result is you can only record yourself. You will only hear one party,and there will be no phone numbers listed.
> 
> Yes as you say google is restricted above things. but let's asume here we need to just for our test/educational purpose we record audio of both side so how to go with that.
> 
> As currenlty i have able to capture only one side of voice call person audio record not able to record both side audio record.
> 
> also there are some apps on google play store they have successfully record audio of both side.

which apps on google play store have successfully recorded audio of both side?

**Comment by savan15** [comment]
App name : Call Recorder - Cube ACR
